### ISSUE TITLE
Content bleeds into popup

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2094]
### Subject of the issue
When a popup is open the rest of the document should be inaccessible to the screen reader.
Currently this works for tabbing but not for the screen reader cursor.

**Comment by oliverfoster** [comment]
merged
